test/integration/cloud-run/cloud_run_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloud_run
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMonitoringAlertSecret(t *testing.T) {
+	vars := map[string]interface{}{
+		"keyring": "test_keyring",
+		"key":     "test_key",
+	}
+
+	secretT := tft.NewTFBlueprintTest(t, tft.WithVars(vars))
+
+	projectId := secretT.GetTFOptions().EnvVars["TF_VAR_project_id"]
```

Comment:
```suggestion
	projectId := bpt.GetStringOutput("project_id")
```

Reply:
I need the project_id before the test starts to run the terraform code, that's why I have to get the project id this way.

Reply:
I put the wrong fix in the comment, sorry. I meant to say:
`projectId := bpt.GetTFSetupJsonOutput("projectId")`